packages/nestjs-slack/README.md

```diff
@@ -0,0 +1 @@
+TBA
```

Comment:
we recommend [standard-readme](https://github.com/RichardLitt/standard-readme) format :)